jwcrypto/jwk.py

```diff
@@ -753,11 +819,11 @@ def import_from_pyca(self, key):
             self._import_pyca_pri_ec(key)
         elif isinstance(key, ec.EllipticCurvePublicKey):
             self._import_pyca_pub_ec(key)
-        elif isinstance(key, ed25519.Ed25519PrivateKey) or \
-                isinstance(key, ed448.Ed448PrivateKey):
+        elif isinstance(key, Ed25519PrivateKey) or \
```

Comment:
isinstance supports tuple of types: ``isinstance(key, (Ed25519PrivateKey, ed448.Ed448PrivateKey)):``

Reply:
ok